src/renderer/dx/DxFontRenderData.cpp

```diff
@@ -642,6 +734,19 @@ CATCH_RETURN()
     BOOL familyExists;
     THROW_IF_FAILED(fontCollection->FindFamilyName(familyName.data(), &familyIndex, &familyExists));
 
+    // If the system collection missed, try the files sitting next to our binary.
+    if (!familyExists)
+    {
+        const auto nearbyCollection = NearbyCollection();

```

Comment:
it's the tiniest nit, but this will reintroduce the addref/release. I also don't _really_ care at this point because it's not on every single font lookup or call to NearbyCollection.

I *think* the correct resolution is `auto&& nearbyCollection = NearbyCollection()`, whiich should derive a const reference automatically? Let me.. test that in godbolt

Reply:
auto&& or const auto& works for this.

Reply:
```suggestion
        auto&& nearbyCollection = NearbyCollection();
```